cedar-policy-core/src/ast/literal.rs

```diff
@@ -125,6 +128,42 @@ impl From<Arc<EntityUID>> for Literal {
     }
 }
 
+#[cfg(feature = "protobufs")]
+impl From<&proto::expr::Literal> for Literal {
+    // PANIC SAFETY: experimental feature
+    #[allow(clippy::expect_used)]
+    fn from(v: &proto::expr::Literal) -> Self {
+        match v.lit.as_ref().expect("as_ref() for field that will exist") {
```

Comment:
Looks like this `expect` is everywhere so please consider my previous comment for all instances.